docs/design/mvp-summary-for-client.md

```diff
@@ -0,0 +1,65 @@
+# MVP Scope Summary for Client
+
+## Identity Access Control
+- User accounts
+  - manual creation, modification, and deletion for MVP
+- Staff roles
+  - manual creation, modification, and deletion for MVP
+- IP whitelist (can only access Chronicle from onsite at spa)
+
+## Customer Profiles
+- Core profiles field - creation and modification
+  - name
+  - address
+  - email
+  - phone number
+  - dob
+  - sex and gender
+- Medical record
+  - Weight and height
+  - Allergies
+  - Conditions
+  - Additional Notes
+- Membership level record
+  - Membership purchase and renewal history
+
+## Entries
+- Activity Entry 
+  - activity name
+  - price
+  - date/time
+  - provider
+  - general notes
+  - critical notes
+  - photos
+- Purchase Entry
+  - product name
+  - variant
+  - price
+  - barcode
+  - associated activity entry (if applicable)
+- client creation, deletion, modification
+- Entries will be associated with customers and providers
+- Log of modifications to entries made by staff
```

Comment:
Is keeping track of modification history part of MVP scope?

Reply:
yes, just a basic log just a list of staff members who have edited a given entry and at what date and time. 

It also includes when the entry was first made.